buckbuild.bzl

```diff
@@ -1727,14 +1727,19 @@ def define_buck_targets(
         name = "mobile_bytecode",
         header_namespace = "",
         exported_headers = {
-            "torch/csrc/jit/serialization/mobile_bytecode_generated.h": ":mobile_bytecode_header[mobile_bytecode_generated.h]",
+            ("torch/csrc/jit/serialization/mobile_bytecode_generated.h" if IS_OSS
```

Comment:
Any reason there is a different macro `IS_OSS` here?

Reply:
Yeah the generated file name changed for fbcode (to mobile_bytecode_generated_fbsource); and was causing a CI failure: https://github.com/pytorch/pytorch/actions/runs/3297660939/jobs/5438753050